crates/fuel-core/src/combined_database.rs

```diff
@@ -71,15 +75,17 @@ impl CombinedDatabase {
         relayer: Database<Relayer>,
         gas_price: Database<GasPriceDatabase>,
         compression: Database<CompressionDatabase>,
-        block_aggregation: Database<BlockAggregatorDatabase>,
+        block_aggregation_storage: Database<BlockAggregatorDatabaseStorage>,
+        block_aggregation_s3: Database<BlockAggregatorDatabaseS3>,
```

Comment:
It should be enough for us to have just one database instance per service. In the case if we switched from one type of storing to another one, we can throw and error saying that the user should clean-up the old database.